app/ui/src/main/res/layout/about.xml

```diff
@@ -1,5 +1,19 @@
 <?xml version="1.0" encoding="utf-8"?>
-<WebView xmlns:android="http://schemas.android.com/apk/res/android"
-    android:id="@+id/about_view"
-    android:layout_width="match_parent"
-    android:layout_height="match_parent" />
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+              android:orientation="vertical"
+              android:layout_height="fill_parent"
+              android:layout_width="fill_parent">
+
+    <android.support.v7.widget.Toolbar
```

Comment:
The same toolbar element appears to be used in every layout that needs one  
How about pulling this out into its own layout, then using `<include layout="@layout/toolbar.xml"/>` here instead?